README.md

```diff
@@ -35,6 +35,27 @@ $ docker build -t my-nodejs-app .
 $ docker run -it --rm --name my-running-app my-nodejs-app
 ```
 
+If you prefer Docker Compose:
+
+```yaml
+version: "2"
+services:
+  node:
+    image: "node:4-onbuild"
```

Comment:
@yosifkit  is right! It only works for me because I have `express` installed locally. I assumed the .`dockerignore` was preventing it. 

What would be the appropriate Docker Compose example? Changing my example above to `image: "node:latest"` doesn't work. Here's a revised `onbuild` Docker Compose example, but it depends on having a Dockerfile.

**docker-compose.yaml**
```yaml
version: "2"
services:
  node:
    build: .
    environment:
      - NODE_ENV=production
    ports:
      - "8080:8080"

```

**Dockerfile**

```
FROM node:4-onbuild

```
**package.json**
```json
{
    "name": "docker-test",
    "version": "1.0.0",
    "description": "",
    "main": "index.js",
    "scripts": {
        "test": "echo \"Error: no test specified\" && exit 1",
        "start": "node index.js"
    },
    "keywords": [],
    "author": "",
    "license": "ISC",
    "dependencies": {
        "express": "^4.14.1"
    }
}

```


**index.js**
```js
var express = require('express')
var app = express()

app.get('/', function (req, res) {
  res.send('Hello World!')
})

app.listen(8080, function () {
  console.log('Example app listening on port 8080!')
})
```

**.dockerignore** 
```
node_modules
```